{
  "channeldata": {
    "activate.d": false,
    "binary_prefix": false,
    "deactivate.d": false,
    "description": null,
    "dev_url": null,
    "doc_source_url": null,
    "doc_url": null,
    "home": "https://github.com/pytest-dev/pytest-faulthandler",
    "icon_hash": null,
    "icon_url": null,
    "identifiers": null,
    "keywords": null,
    "license": "MIT",
    "post_link": false,
    "pre_link": false,
    "pre_unlink": false,
    "recipe_origin": null,
    "run_exports": {},
    "source_git_url": null,
    "source_url": "https://pypi.io/packages/source/p/pytest-faulthandler/pytest-faulthandler-1.3.0.zip",
    "subdirs": [
      "win-64"
    ],
    "summary": "py.test plugin that activates the fault handler module for tests",
    "tags": null,
    "text_prefix": false,
    "timestamp": 0,
    "version": "1.3.0"
  },
  "channeldata_version": 1,
  "feedstock": null,
  "labels": [
    "main",
    "cf202003"
  ],
  "package": "pytest-faulthandler-1.3.0-py27_0.tar.bz2",
  "repodata": {
    "build": "py27_0",
    "build_number": 0,
    "depends": [
      "pytest",
      "python 2.7*"
    ],
    "license": "MIT",
    "md5": "8838847792e6a7cce4874bd64b84cfad",
    "name": "pytest-faulthandler",
    "sha256": "ce5802aee91a37cca1c07823b40bf1259ae395ebd3138c000e23450d4f07bcec",
    "size": 4297,
    "subdir": "win-64",
    "version": "1.3.0"
  },
  "repodata_version": 1,
  "subdir": "win-64",
  "url": "https://conda.anaconda.org/conda-forge/win-64/pytest-faulthandler-1.3.0-py27_0.tar.bz2"
}